src/CampaignAdmin.php

```diff
@@ -193,7 +193,7 @@ protected function getListResource()
     protected function getChangeSetResource(ChangeSet $changeSet)
     {
         $hal = [
-            '_links' => [
```

Comment:
This might make sense for react.js , but it breaks HAL which this the format this code is written for.

http://stateless.co/hal_specification.html